### PR TITLE
#847 Stocktake list view using new API

### PR DESCRIPTION
**Pull request by joshxg**
#847 

- This is just getting the stocktakes list view using the new types and queries/api

### CODE REVIEW

File: packages/inventory/src/Stocktake/ListView/AppBarButtons.tsx [hunk 1]
```diff
@@ -25,7 +20,7 @@ export const AppBarButtons: FC<AppBarButtonsProps> = ({ onCreate }) => {
         <ButtonWithIcon
           Icon={<PlusCircleIcon />}
           label={t('label.new-stocktake')}
-          onClick={() => onCreate(true)}
+          onClick={() => {}}
```
Comment (joshxg) [inline packages/inventory/src/Stocktake/ListView/AppBarButtons.tsx:23]: Have just removed this for now as the detail view wasn't working

File: packages/inventory/src/Stocktake/ListView/ListView.tsx [hunk 2]
```diff
@@ -64,30 +48,16 @@ export const StocktakeListView: FC = () => {
...
-      );
-      errorSnack();
-    }
-  };
-
   return (
     <>
-      <Toolbar onDelete={onDelete} data={data} filter={filter} />
-      <AppBarButtons onCreate={onNewStocktake} />
+      <Toolbar onDelete={() => {}} data={data?.nodes} filter={filter} />
```
Comment (joshxg) [inline packages/inventory/src/Stocktake/ListView/ListView.tsx:53]: will create a delete hook as a specific issue - really just wanting to get into a compiling state

**Review by joshxg — COMMENTED** [review]
I have created a stocktake for the store `Opua` on the demo server, which you can see if you log into that store and go to the stocktakes list view

**Review by mark-prins — APPROVED** [review]
👍  very nice

**Pull request merged**